.github/workflows/publish.yml

```diff
@@ -34,6 +34,8 @@ jobs:
     - run: yarn workspace @department-of-veterans-affairs/web-components run build
     - run: yarn workspace @department-of-veterans-affairs/web-components run build-bindings
     - run: yarn workspace @department-of-veterans-affairs/component-library run build
+    - run: yarn workspace @department-of-veterans-affairs/css-library run build:tokens
+    - run: yarn workspace @department-of-veterans-affairs/css-library run build:minify
```

Comment:
These steps will build the `css-library` package so that it can be published. In a separate PR I will make the package public and update the version so that we can tag a pre-release off of the PR branch.